samtranslator/model/apigateway.py

```diff
@@ -20,7 +20,8 @@ class ApiGatewayRestApi(Resource):
             'Parameters': PropertyType(False, is_type(dict)),
             'EndpointConfiguration': PropertyType(False, is_type(dict)),
             "BinaryMediaTypes": PropertyType(False, is_type(list)),
-            "MinimumCompressionSize": PropertyType(False, is_type(int))
+            "MinimumCompressionSize": PropertyType(False, is_type(int)),
+            'OpenApiVersion': PropertyType(False, is_str())
```

Comment:
Remove this line. This is the ApiGateway resource (maps directly to the CFN equivalent). The OpenApiVersion flag only needs to be added to the `Serverless::Api` resource found in the `sam_resources.py` file.

Reply:
will do thanks!